### PR TITLE
Adds Basic Styles

### PR DESCRIPTION
**Pull request by gemfarmer**
For #12
Added basic scss framework, borrowing setup from [tock](https://github.com/18F/tock)
It is using a [Bourbon Neat](http://neat.bourbon.io/) grid layout, similar to several other 18F projects.

screen shot 2015-09-21 at 11 07 45 pm

### CODE REVIEW

**Comment by gemfarmer** [comment]
Includes view/controller for displaying all employees

**Comment by jessieay** [comment]
BEAUTIFUL!

File: Gemfile [hunk 1]
```diff
@@ -10,6 +10,7 @@ gem "rails", "~> 4.2.0"
 gem "simple_form"
 gem "slack-ruby-client"
 gem "uglifier"
+gem 'sass-rails'
```
Comment (jessieay) [inline Gemfile]: to stay consistent - double quotes here?

File: app/assets/javascripts/application.js [hunk 2]
```diff
@@ -1,3 +1,3 @@
 //= require jquery
 //= require jquery_ujs
-//= require_tree .
+//= require_tree .
```
Comment (jessieay) [inline app/assets/javascripts/application.js]: no newline at EOF! Consider adding this as a default to sublime (I think that's the editor you use?) http://www.sublimetext.com/forum/viewtopic.php?f=3&t=12118

File: app/assets/stylesheets/application.scss [hunk 3]
```diff
@@ -0,0 +1,16 @@
+@import "sass/core";
+
+// body{
```
Comment (jessieay) [inline app/assets/stylesheets/application.scss]: remove commented stuff?

File: app/assets/stylesheets/application.scss [hunk 4]
```diff
@@ -0,0 +1,16 @@
...
+// 	background: url("dolores.jpg");
+// 	background-size: 100%;
+// }
+// .container{
+// 	background: white;
+// 	padding: 50px;
+// }
+
+.logo-18f{
+	width: 25px;
```
Comment (jessieay) [inline app/assets/stylesheets/application.scss]: is this standard indentiation for css? I usually see a space before `{` and 2 spaces for indentation - seems like that is also the preferred 18F style https://github.com/18F/frontend/blob/18f-pages-staging/pages/css-styleguide/format.md

File: app/assets/stylesheets/application.scss [hunk 5]
```diff
@@ -0,0 +1,16 @@
...
+	float: left;
+	margin-right: 10px;
+}
```
Comment (jessieay) [inline app/assets/stylesheets/application.scss]: no newline at EOF

File: app/assets/stylesheets/sass/_alert.scss [hunk 6]
```diff
@@ -0,0 +1,10 @@
+.alert {
+	-webkit-box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
+	box-shadow: 0 1px 2px rgba(0, 0, 0, 0.05);
+	background-color: #d0e9c6;
+	font-size: 0.8rem;
+	padding: 0.5em;
+	@include span-columns(10);
+	border-radius:0.5em;
+	margin-bottom: 1em;
+}
```
Comment (jessieay) [inline app/assets/stylesheets/sass/_alert.scss]: no newline at EOF

File: app/assets/stylesheets/sass/_banner.scss [hunk 7]
```diff
@@ -0,0 +1,3 @@
+.banner {
+	@include span-columns(2);
+}
```
Comment (jessieay) [inline app/assets/stylesheets/sass/_banner.scss]: EOF

File: app/assets/stylesheets/sass/_content.scss [hunk 8]
```diff
@@ -0,0 +1,3 @@
+.content {
+	@include span-columns(10);
+}
```
Comment (jessieay) [inline app/assets/stylesheets/sass/_content.scss]: EOF

File: app/assets/stylesheets/sass/_custom.scss [hunk 9]
```diff
@@ -0,0 +1,5 @@
+.logo-18f{
+	width: 25px;
+	float: left;
+	margin-right: 10px;
+}
```
Comment (jessieay) [inline app/assets/stylesheets/sass/_custom.scss]: EOF

File: app/assets/stylesheets/sass/_utils.scss [hunk 10]
```diff
@@ -0,0 +1,3 @@
+.icon-18f {
+	color: #1188ff;
+}
```
Comment (jessieay) [inline app/assets/stylesheets/sass/_utils.scss]: EOF

**Pull request merged**